api/file-translations/index.ts

```diff
@@ -120,4 +147,23 @@ export class SmartlingFileTranslationsApi extends SmartlingBaseApi {
         delete headers["content-type"];
         return headers;
     }
+
+    private static async downloadResponseToTranslatedFileDto(
+        response: Response
+    ): Promise<TranslatedFileDto> {
+        const contentType = response.headers.get("content-type") ?? undefined;
+        const contentDisposition = response.headers.get("content-disposition");
+        let fileName;
+        if (contentDisposition) {
+            const fileNameMatch = contentDisposition.match(/filename="?([^"]+)"?/);
```

Comment:
With this regexp this string `attachment; filename=test\.xml test` will match `test\.xml test` as a filename (because your regexp assumes there might be no quetoes at all). Also, is filename **always** surrounded by doublequostes? From this string `attachment; filename='test\.xml' test`filename will be matched as `'test\.xml' test` (not sure if filename is the last property in the content disposition).

I might be missing some HTTP standard that described quotes vs no quotes vs doublequotes etc. Just want to double check here this regexp will cover all the cases/responses from our api.

Reply:
I've checked the code of the API returning files: it always uses following format: 
```
ATTACHMENT_TEMPLATE = "attachment; filename=\"%s\""; 
```
so, it always uses quotes (`"`). I've changed the regex to find a file name inside quotes and to cover case with escaped quote inside.